std/conv.d

```diff
@@ -1883,6 +1883,49 @@ if (isInputRange!S && !isInfinite!S && isSomeChar!(ElementEncodingType!S) &&
     assert(to!int(byDchar("10"), 10) == 10);
 }
 
+/**
+String, or string-like input range, to char type not directly
```

Comment:
Sorry to jump in after this is already been merged,  but this documentation will not be seen because this function is private. This should have gone as the docs to an example underneath to.

Reply:
Hm... the docs don't talk about `to!char(string)` which worked before this change (and was the impetus to finding this limitation). I suppose it wouldn't hurt to rearrange the unit test to be part of the public docs and add some more comments.

Reply:
The thing is, since all `to` does is call `toImpl`, all documentation of the latter does not show up in the generated docs. As a matter of fact, the unit tests I added do not call `to` directly, but rather `toImpl`, so there is not much sense in having them document `to`. But I do agree `to` could benefit from better documentation. Should that be a separate issue?

Reply:
Sounds good

Reply:
@Dechcaudron would you mind refactoring your unit tests to call `to` instead of `toImpl` (I should have caught that), moving the unit test to be under the `to` function, making it documented and putting a little blurb about it? I think you could even add calls for `to!char(string)` to show that works also.

Reply:
I think it makes sense to duplicate the tests rather than to plain move them. After all, we still want this particular override of `toImpl` tested. If I'm going to do something that affects the public documentation of `to`, I want to put some more thought into it. I'll get back to you guys about this in some days. I need to disconnect from Phobos a little. 

Should I create a separate pull request once I've gone about the above? Shall we create a new issue then?

Reply:
Guys, ready to do this over here. New issue, or just a standalone PR?

Reply:
just make a new PR, no need for a bugzilla issue